### PR TITLE
Seometa + contact page change + service page change

### PR DESCRIPTION
**Pull request by paulchrisluke**
im adding some head stuff in prep to follow meta addition guides

### CODE REVIEW

**Comment by paulchrisluke** [comment]
note, I've linked partner page images in this PR as well

**Comment by paulchrisluke** [comment]
https://github.com/pcl-labs/vodworks-storyblok/issues/11 completed

File: pages/partner/index.vue [hunk 1]
```diff
@@ -154,64 +164,81 @@
...
       <div class="pt-20 grid lg:grid-cols-4 px-10 lg:px-6 gap-8">
         <!-- card -->
-        <div class="p-4 bg-b-dark-gray rounded-2xl text-white text-center">
-          <!-- image -->
-          <div
-            class="w-full h-56 bg-center bg-no-repeat bg-cover rounded-tr-2xl rounded-tl-2xl"
-            :style="resolveBackground('/img/partners/img_invest_1.png')"
-          ></div>
+        <NuxtLink
+          to="https://phnompenh.impacthub.net/khmer-tourism-for-the-future/?utm_source=vodworks&utm_medium=referral&utm_campaign=our-partners-page"
```
Comment (giteshk11) [inline pages/partner/index.vue]: If its an external link please replace NuxtLink with `<a href>`

File: pages/partner/index.vue [hunk 2]
```diff
@@ -154,64 +164,81 @@
...
+        </NuxtLink>
 
         <!-- card -->
-        <div class="p-4 bg-b-dark-gray rounded-2xl text-white text-center">
-          <!-- image -->
-          <div
-            class="w-full h-56 bg-center bg-no-repeat bg-cover rounded-tr-2xl rounded-tl-2xl"
-            :style="resolveBackground('/img/partners/img_invest_2.png')"
-          ></div>
+        <NuxtLink
```
Comment (giteshk11) [inline pages/partner/index.vue]: Same here as well

File: pages/partner/index.vue [hunk 3]
```diff
@@ -154,64 +164,81 @@
...
+        >>
 
         <!-- card -->
-        <div class="p-4 bg-b-dark-gray rounded-2xl text-white text-center">
-          <!-- image -->
-          <div
-            class="w-full h-56 bg-center bg-no-repeat bg-cover rounded-tr-2xl rounded-tl-2xl"
-            :style="resolveBackground('/img/partners/img_invest_3.png')"
-          ></div>
+        <NuxtLink
```
Comment (giteshk11) [inline pages/partner/index.vue]: Here also

File: pages/partner/index.vue [hunk 4]
```diff
@@ -154,64 +164,81 @@
...
+        </NuxtLink>
 
         <!-- card -->
-        <div class="p-4 bg-b-dark-gray rounded-2xl text-white text-center">
-          <!-- image -->
-          <div
-            class="w-full h-56 bg-center bg-no-repeat bg-cover rounded-tr-2xl rounded-tl-2xl"
-            :style="resolveBackground('/img/partners/img_invest_4.png')"
-          ></div>
+        <NuxtLink
```
Comment (giteshk11) [inline pages/partner/index.vue]: Here also

**Review by giteshk11 — CHANGES_REQUESTED** [review]

**Pull request merged**